pcsx2/gui/Panels/SpeedhacksPanel.cpp

```diff
@@ -23,16 +23,24 @@ const wxChar* Panels::SpeedHacksPanel::GetEEcycleSliderMsg( int val )
 {
 	switch( val )
 	{
-		case 1:
-			return pxEt( L"1 - Default cyclerate. This closely matches the actual speed of a real PS2 EmotionEngine."
+		case -2:
+			return pxEt( L"-2 - Increases the EE's cyclerate by about 50%. Greatly increases hardware requirements, may noticeably increase in-game FPS. \n This setting can cause games to FAIL TO BOOT."
```

Comment:
Minor complaint - spaces around `\n` are unnecessary.

Apart from that, GUI has been tested.